File: [프로그래머스] 연습문제_과일 장수.py
def solution(k, m, score):
    answer = 0
    arr = []
    score.sort(reverse = True)
    for i in range(0, len(score), m):
        arr = score[i:i+m]
        if len(arr) == m:
            answer += min(arr) * m
    return answer

-----------------------------

def solution(k, m, score):
    answer = 0
    score.sort(reverse = True)
    for i in range(1, len(score)//m + 1):
        answer += score[m * i - 1] * m
    return answer

File: [프로그래머스] 연습문제_과일 장수.js
function solution(k, m, score) {
    var answer = 0;
    apple = score.sort((a, b) => b - a)
    let i = 0
    while(true){
        if(apple.slice(i, i + m).length < m){
            return answer;
        }
        answer += apple.slice(i, i + m).at(-1) * m
        i += m
    }
}